### PR TITLE
Changes in tutorial

### PR DESCRIPTION
**Pull request by AnnikaGonnermann**
- exchanged screenshot
- "free trial" > "trial"
- added links to core documentation

### CODE REVIEW

File: tutorials/btp-cockpit-entitlements/btp-cockpit-entitlements.md [hunk 1]
```diff
@@ -1,77 +1,77 @@
...
-
-
----
+---
+title: Manage Entitlements Using the Cockpit
+description: Configure entitlements and quota using the cockpit. (Free Tier ready)
+auto_validation: true
+time: 15
+tags: [ tutorial>beginner, topic>cloud, software-product-function>sap-btp-cockpit, tutorial>free-tier]
+primary_tag: products>sap-business-technology-platform
```
Comment (MichaelCzcz) [inline tutorials/btp-cockpit-entitlements/btp-cockpit-entitlements.md:7]: Please correct tag (products -> software-product)

File: tutorials/btp-cockpit-setup/btp-cockpit-setup.md [hunk 2]
```diff
@@ -1,78 +1,78 @@
...
-
-
-
-
----
+---
+title: Get an SAP BTP Account for Tutorials
+description: Set up an SAP Business Technology Platform account so you can do the tutorials involving SAP BTP services and applications (trial or free tier).
+time: 5
+tags: [ tutorial>beginner, products>sap-business-technology-platform, tutorial>type>free-tier]
```
Comment (MichaelCzcz) [inline tutorials/btp-cockpit-setup/btp-cockpit-setup.md:5]: Please correct tags (products -> software-product)

File: tutorials/btp-cockpit-setup/btp-cockpit-setup.md [hunk 3]
```diff
@@ -1,78 +1,78 @@
...
+primary_tag: products>sap-business-technology-platform
```
Comment (MichaelCzcz) [inline tutorials/btp-cockpit-setup/btp-cockpit-setup.md:6]: Please correct tag (products -> software-product)

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

File: tutorials/btp-cockpit-setup/btp-cockpit-setup.md (hunk 3 above)
Comment (AnnikaGonnermann) [reply tutorials/btp-cockpit-setup/btp-cockpit-setup.md:6]: Hey @MichaelCzcz  - sorry, I do not own these tutorials I just adapted some steps in reference to the documentation. Can you share more details what's wrong with the tags?
Annika

File: tutorials/btp-cockpit-entitlements/btp-cockpit-entitlements.md (hunk 1 above)
Comment (AnnikaGonnermann) [reply tutorials/btp-cockpit-entitlements/btp-cockpit-entitlements.md:7]: Hey @MichaelCzcz  - sorry, I do not own these tutorials I just adapted some steps in reference to the documentation. Can you share more details what's wrong with the tags?
Annika

File: tutorials/btp-cockpit-setup/btp-cockpit-setup.md (hunk 2 above)
Comment (AnnikaGonnermann) [reply tutorials/btp-cockpit-setup/btp-cockpit-setup.md:5]: Hey @MichaelCzcz  - sorry, I do not own these tutorials I just adapted some steps in reference to the documentation. Can you share more details what's wrong with the tags?
Annika

File: tutorials/btp-cockpit-setup/btp-cockpit-setup.md (hunk 3 above)
Comment (MichaelCzcz) [reply tutorials/btp-cockpit-setup/btp-cockpit-setup.md:6]: The name space is invalid. It has changed from products to software-product

Comment (AnnikaGonnermann) [reply tutorials/btp-cockpit-setup/btp-cockpit-setup.md]: Thanks for clarifying - Changed 👍

**Pull request merged**